src/App.js

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
-import './App.css';
+import Bookslist from './components/BooksList';
```

Comment:
Let's do some rearrangement in here.

Kindly move `app.js` into `src/component` dir

![image](https://user-images.githubusercontent.com/31897434/90045002-57080480-dcc6-11ea-8c39-129c865acf05.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/